src/app/activity/bounty/Create.ts

```diff
@@ -1,145 +1,187 @@
+
+        await updateMessageStore(newBounty, message);
+    
+        await message.react('💰');
+        return await message.react('❌');
+    
+    } else {
 
-    const publishOrDeleteMessage = 
-        'Thank you! If it looks good, please hit 👍 to publish the bounty.\n' +
-        'Once the bounty has been published, others can view and claim the bounty.\n' +
-        'If you are not happy with the bounty, hit ❌ to delete it and start over.\n'
-    await guildMember.send(publishOrDeleteMessage);
-    const message: Message = await guildMember.send(bountyPreview);
+        let bountyPreview: MessageOptions = {
+            embeds: [{
+                title: await BountyUtils.createPublicTitle(newBounty),
+                url: (process.env.BOUNTY_BOARD_URL + newBounty._id),
+                author: {
+                    icon_url: guildMember.user.avatarURL(),
+                    name: `${newBounty.createdBy.discordHandle}: ${guildId}`,
+                },
+                description: newBounty.description,
+                fields: [
+                    // TODO: figure out a way to explicitly match order with BountyEmbedFields
+                    // static bountyId = 0;
+                    // static criteria = 1;
+                    // static reward = 2;
+                    // static status = 3;
+                    // static deadline = 4;
+                    // static createdBy = 5;
+                    { name: 'Bounty Id', value: newBounty._id.toString(), inline: false },
+                    { name: 'Criteria', value: newBounty.criteria.toString() },
+                    { name: 'Reward', value: newBounty.reward.amount + ' ' + newBounty.reward.currency, inline: true },
+                    { name: 'Status', value: BountyStatus.open, inline: true },
+                    { name: 'Deadline', value: BountyUtils.formatDisplayDate(newBounty.dueAt), inline: true },
+                    { name: 'Created by', value: newBounty.createdBy.discordHandle.toString(), inline: true },
+                ],
+                timestamp: new Date().getTime(),
+                footer: {
+                    text: '👍 - publish | ❌ - delete | Please reply within 60 minutes',
+                },
+            }],
+        };
 
-    await updateMessageStore(newBounty, message);
+        const publishOrDeleteMessage = 
+            'Thank you! If it looks good, please hit 👍 to publish the bounty.\n' +
+            'Once the bounty has been published, others can view and claim the bounty.\n' +
+            'If you are not happy with the bounty, hit ❌ to delete it and start over.\n'
+        await guildMember.send(publishOrDeleteMessage);
+        const message: Message = await guildMember.send(bountyPreview);
 
-    await message.react('👍');
-    return await message.react('❌');
-}
+        await updateMessageStore(newBounty, message);
 
+        await message.react('👍');
+        return await message.react('❌');
```

Comment:
Do we still need these reaction callbacks given we have the reaction event listener?

Reply:
Maybe, but if it ain't broke...? I think the downside of taking it out is the user will see a mysterious reaction happen after the card appears.